src/b3d/chisight/gen3d/settings.py

```diff
@@ -17,10 +17,10 @@
 
 p_resample_color = 0.005
 hyperparams = {
-    "pose_kernel": transition_kernels.UniformPoseDriftKernel(max_shift=0.2),
+    "pose_kernel": transition_kernels.GaussianVMFPoseDriftKernel(0.02, 1000.0),
     "color_kernel": transition_kernels.MixtureDriftKernel(
         [
-            transition_kernels.LaplaceNotTruncatedColorDriftKernel(scale=0.05),
+            transition_kernels.RenormalizedLaplaceColorDriftKernel(scale=0.05),
```

Comment:
I switched to this just since I was wondering if weird issues could arise due to not having a normalized distribution, but we can revert it if you'd like.  It didn't visibly change performance in my experiments.